integration-tests/src/test/scala/tech/beshu/ror/integration/suites/ResponseFieldRuleSuite.scala

```diff
@@ -0,0 +1,75 @@
+/*
+ *    This file is part of ReadonlyREST.
+ *
+ *    ReadonlyREST is free software: you can redistribute it and/or modify
+ *    it under the terms of the GNU General Public License as published by
+ *    the Free Software Foundation, either version 3 of the License, or
+ *    (at your option) any later version.
+ *
+ *    ReadonlyREST is distributed in the hope that it will be useful,
+ *    but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *    GNU General Public License for more details.
+ *
+ *    You should have received a copy of the GNU General Public License
+ *    along with ReadonlyREST.  If not, see http://www.gnu.org/licenses/
+ */
+package tech.beshu.ror.integration.suites
+
+import org.scalatest.{Matchers, WordSpec}
+import tech.beshu.ror.integration.suites.base.support.BaseSingleNodeEsClusterTest
+import tech.beshu.ror.utils.containers.{ElasticsearchNodeDataInitializer, EsContainerCreator}
+import tech.beshu.ror.utils.elasticsearch.{CatManager, ClusterManager, DocumentManager, SearchManager}
+import tech.beshu.ror.utils.httpclient.RestClient
+
+trait ResponseFieldRuleSuite
+  extends WordSpec
+    with BaseSingleNodeEsClusterTest
+    with Matchers {
+  this: EsContainerCreator =>
+
+  override implicit val rorConfigFileName = "/response_field_rules/readonlyrest.yml"
+
+  override def nodeDataInitializer = Some(ResponseFieldRuleSuite.nodeDataInitializer())
+
+  "A response_field rule" should {
```

Comment:
oh, it'd be nice to check if the filtering works also for different format (eg. `?format=yaml`)

Reply:
I tested it manually before, but indeed automated test would be better